PalasoUIWindowsForms/Reporting/MemoryManagement.cs

```diff
@@ -45,9 +45,11 @@ public static bool CheckMemory(bool minor, string eventDescription, bool okToBot
 			string totalVirtualMemory = "unknown";
 			if (Platform.IsWindows)
 			{
+#if !MONO
```

Comment:
generally prefer __MonoCS__ to MONO.

I suppose we could dispense with the using statement altogether and attempt reflection here.  (with a comment explaining the issue)  if we do that, it's possible that we don't need the reference in the csproj file, but I'm not sure about that.